tests/Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "MIT"
 publish = false
 
 [dev-dependencies]
-env_logger = "0.8"
```

Comment:
oops. I wish there was a CI check for dupes...

Reply:
we could use `cargo deny` for but it will generate a bunch of errors/warnings because of the `benches with jsonrpc crate` and `your added example  but maybe we could feature gate it instead.